sdk/storage/storage-file-share/test/fileclient.spec.ts

```diff
@@ -3040,40 +3041,41 @@ describe("FileClient - NFS", () => {
       fileType: "Regular",
     };
     const cResp = await fileClient.create(content.length);
-    assert.deepEqual(cResp.owner, "0");
-    assert.deepEqual(cResp.group, "0");
-    assert.ok(cResp.fileMode);
-    assert.ok(cResp.nfsFileType);
+    assert.deepEqual(cResp.posixProperties?.owner, "0");
+    assert.deepEqual(cResp.posixProperties?.group, "0");
+    assert.ok(cResp.posixProperties?.fileMode);
+    assert.ok(cResp.posixProperties?.fileType);
 
     const setResp = await fileClient.setProperties({ posixProperties });
-    assert.deepEqual(setResp.owner, posixProperties.owner);
-    assert.deepEqual(setResp.group, posixProperties.group);
-    assert.deepEqual(setResp.fileMode, posixProperties.fileMode);
+    assert.deepEqual(setResp.posixProperties?.owner, posixProperties.owner);
+    assert.deepEqual(setResp.posixProperties?.group, posixProperties.group);
+    assert.deepEqual(setResp.posixProperties?.fileMode, posixProperties.fileMode);
+    assert.deepEqual(setResp.posixProperties?.linkCount, 1);
```

Comment:
Please check if set return link count.

Reply:
It returns link count, we have handled it in setProperties function definition.